GroovyDataCleanerExtension/pom.xml

```diff
@@ -36,9 +36,9 @@
 			<version>2.0.5</version>
 		</dependency>
 		<dependency>
-			<groupId>org.eobjects.analyzerbeans</groupId>
-			<artifactId>AnalyzerBeans-core</artifactId>
-			<version>0.22</version>
+			<groupId>org.eobjects.datacleaner</groupId>
+			<artifactId>DataCleaner-engine-core</artifactId>
+			<version>4.0-SNAPSHOT</version>
```

Comment:
It is available in central repo so you should be able to convince your stubborn eclipse with a compelling argument [1] ;-)

[1] http://repo1.maven.org/maven2/org/eobjects/datacleaner/DataCleaner-engine-core/4.0-RC5/